helm/jaeger-operator-app/README.md

```diff
@@ -48,24 +48,34 @@ The command removes all the Kubernetes components associated with the chart and
+| `resources`                                  | K8s pod resources                                                                                           | `None`                          |
+| `nodeSelector`                               | Node labels for pod assignment                                                                              | `{}`                            |
+| `tolerations`                                | Toleration labels for pod assignment                                                                        | `[]`                            |
+| `affinity`                                   | Affinity settings for pod assignment                                                                        | `{}`                            |
+| `securityContext`                            | Security context for pod                                                                                    | `{}`                            |
+| `collector.service.enabled`                  | creates the collector service on admin-http port 14269                                                      | `false`                       |
+| `collector.serviceMonitor.enabled`           | creates the corresponding service monitor for collector metrics                                             | `false`                       |
+| `collector.serviceMonitor.additionalLabels`  | additional labels for collector service monitor                                                             | `{}`                            |
+| `collector.serviceMonitor.interval`          | Interval at which metrics should be scraped                                                                 | `10s`                           |
+| `collector.serviceMonitor.scrapeTimeout`     | Timeout after which the scrape is ended                                                                     | `10s`                           |
+| `collector.serviceMonitor.scheme`            | Scheme to use for scraping                                                                                  | `http`                        |
+| `collector.serviceMonitor.relabelings`       | Relabel configuration for the metrics                                                                       | `[]`                            |
+| `collector.serviceMonitor.targetLabels`      | Set of labels to transfer on the Kubernetes Service onto the target                                         | `[]`                            |
+| `collector.serviceMonitor.metricRelabelings` | MetricRelabelConfigs to apply to samples before ingestion                                                   | `[]`                            |
+| `collector.serviceMonitor.sampleLimit`       | Number of samples that will fail the scrape if exceeded                                                     | `[]`                            |
```

Comment:
I really like that you went through the trouble to document the function of your fields :+1: